review/fileToReview.js

```diff
@@ -11,36 +10,44 @@ var domain = "bank.local.fr"
  * @param {Object} previousTransactions Previous page of transactions (To ckeck for dupes)
  * @return {Object} All transactions available on the page
  */
-async function fetchTransactions(fromDate, authorization, jws = null, id, page, previousTransactions) {
-	console.log(`--- Fetch Trasactions page n°${page} ---`);
-	try {
-    var headers = {"Authorisation":  authorization }
+async function fetchTransactions(
+  fromDate,
+  authorization,
+  jws = null,
+  id,
+  page,
+  previousTransactions
+) {
+  console.log(`--- Fetch Transactions page n°${page} ---`);
+  try {
+    let headers = { Authorization: authorization };
 
     if (jws) {
       headers = {
-        "Authorisation": authorization,
-        "jws": jws,
-        "Content-type": "application/json",
-        "Accept": "application/json"
-      }
+        Authorization: authorization,
+        jws: jws,
+        'Content-type': 'application/json',
+        Accept: 'application/json',
+      };
     } else {
       headers = {
-        "Authorisation": authorization,
-        "Content-type": "application/json",
-        "Accept": "application/json",
-      }
+        Authorization: authorization,
+        'Content-type': 'application/json',
+        Accept: 'application/json',
+      };
     }
 
-	  var {code, response } = await doRequest('GET',
```

Comment:
* var -> let because response form change later in the code 
* doRequest is never called